aten/src/ATen/native/Convolution.cpp

```diff
@@ -742,6 +742,10 @@ static std::tuple<Tensor, bool> batchify(
     const Tensor& input,
     const int64_t num_spatial_dims,
     const std::string& func_name) {
+  // assume NTs are always batched
+  if (input.is_nested()) {
+    return std::make_tuple(input, true);
+  }
```

Comment:
just to give an alternative: You could override `conv2d_transpose` for the `AutogradNestedTensor` key using `py_impl`. But maybe this is more of a pain, since it would require you to re-implement more of `conv2d_transpose` in python?

Reply:
ah thanks for the suggestion! Indeed this would be more work, as I'd have to implement more of `conv2d_transpose`, alongside the other 1d/2d/3d transposed / non-transposed variants.